src/promptflow/promptflow/_constants.py

```diff
@@ -17,6 +17,7 @@
 ERROR_RESPONSE_COMPONENT_NAME = "promptflow"
 EXTENSION_UA = "prompt-flow-extension"
 LANGUAGE_KEY = "language"
+ENABLE_MULTI_CONTAINER_KEY = "EnableMultiContainer"
```

Comment:
we should follow name of other env key, starts with PF_

Reply:
updated